### DISCUSSION TITLE
Setup DNS over HTTPS (DOH) on Firefox Android

### DISCUSSION THREAD

**Discussion by SSUPII** [abskmj/notes#25] [General]
Suggested steps seem to do nothing on Nightly for Android 130, adguard fails and despite mode 3 being used it is still using something else.

**Comment by abskmj** [comment]
The nightly version of the firefox might be unstable.

I use Fennec from [fdroid](https://f-droid.org/packages/org.mozilla.fennec_fdroid/) and current version is v129.0.0. I have set it up with Cloudflare DOH and it is working. I now use uBlock Origin extension to block ads instead of AdGaurd DNS.
